File: livingrimoire start here/LivinGrimoire java/LivinGrimoire/DiHelloWorld.java
package LivinGrimoire;

public class DiHelloWorld extends DiSkillV2{
    // hello world skill for testing purposes
    public DiHelloWorld() {
        super();
    }

    @Override
    public void input(String ear, String skin, String eye) {
        switch (ear){
            case "hello":
                super.setVerbatimAlg(4,"hello world");
                break;
        }
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/livinGrimoire/DiHelloWorld.kt
package livinGrimoire

class DiHelloWorld  // hello world skill for testing purposes
    : DiSkillV2() {
    override fun input(ear: String, skin: String, eye: String) {
        when (ear) {
            "hello" -> super.setVerbatimAlg(4, "hello world") // 1->5 1 is the highest algorithm priority
        }
    }
}